.travis.yml

```diff
@@ -0,0 +1,36 @@
+language: julia
+
+os:
+  - linux
+
+julia:
+  - 1.1
+  - nightly
+
+# Uncomment the following lines to allow failures on nightly julia
+# (tests will run but not make your overall status red)
+matrix:
+  allow_failures:
+  - julia: nightly
+
+notifications:
+  email: false
```

Comment:
Do we not want to be notified?

Reply:
I think no because the builds are so quick. It's not like we'll need email notifications hours later when a build finishes. If you disagree I can enable it.